src/menu_nbgl.c

```diff
@@ -2,9 +2,9 @@
 #include "menu.h"
 #include "nbgl_use_case.h"
 
-#define SETTING_INFO_NB 3
-static const char* const infoTypes[SETTING_INFO_NB] = {"Version", "Developer", "Copyright"};
-static const char* const infoContents[SETTING_INFO_NB] = {APPVERSION, "Jspada", "(c) 2024 Ledger"};
+#define SETTING_INFO_NB 4
+static const char* const infoTypes[SETTING_INFO_NB] = {"Version", "Developer", "Modifications by", "Copyright"};
+static const char* const infoContents[SETTING_INFO_NB] = {APPVERSION, "jspada", "Zondax AG", "(c) 2024 Ledger"};
```

Comment:
We changed the capital letter on the developer item. Besides, this is just flex/stax. What about nano devices?

Reply:
They didn't have any information on the main menu, should we add it there aswell?

Reply:
Fixed this and added Mani Menu Info for the rest of devices

Reply:
Fixed the capitalization and added main menu info for all devices